seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -173,7 +173,7 @@ angular.module('BE.seed.controller.inventory_list', [])
             inventory_type: function () {
               return $stateParams.inventory_type;
             },
-            single_record: function () {
+            provided_inventory: function () {
```

Comment:
@nllong had a typo in there - thanks for catching this!
https://github.com/SEED-platform/seed/pull/2035/files#diff-0aab274639148dc7bdec29d433d9150dR176

Reply:
Cool. I’ll test again